desktop/core/src/desktop/js/reactComponents/AlertComponent/AlertComponent.scss

```diff
@@ -0,0 +1,84 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import '../../components/styles/variables';
+
+.flash-messages {
+    position: fixed;
+    width: 30%;
+    z-index: 9999;
+    top: 7px;
+    right: 0;
+    margin-right: 24px;
+    display: flex;
+    flex-flow: column;
+    row-gap: 10px;
+
+    button:hover,
+    button:focus {
+        background-color: unset;
+    }
+
+    button span {
+        font-size: 12px;
+        padding-top: 5px;
+    }
+
+    button svg.cdp-icon-close {
+        height: 16px;
+    }
+
+    .ant-alert-warning {
+        color: $cui-orange-750;
+        background: $cui-orange-025;
+        border: 1px solid $cui-orange-450;
+
+        button {
+            color: $cui-green-650;
+            ;
+        }
+
+    }
+
+    .ant-alert-success {
+        color: $cui-green-850;
+        background: $cui-green-075;
+        border: 1px solid $cui-green-450;
+
+        button {
+            color: $cui-green-650;
+        }
+    }
+
+    .ant-alert-error {
```

Comment:
Check if it is possible to pass in our own classes to the alert component. It is better to use something like `hue-alert--error` than to overwrite 3rd party names that might change in the future.

Reply:
Should I change it only here or at the rest places also on this file? For example: 
`.ant-alert-warning`, `.ant-alert-success`, etc.
To replace these with `.hue-alert--warning` for example

Reply:
I would say yes if possible